### ISSUE TITLE
Python 3.7 EOL

### ISSUE THREAD

**Issue by MLKaufman** [deeptools/pyGenomeTracks#464]
Great work on the tool and the visualizations!
Python 3.7 has reached end of life this year.  Also conda has also recently depreciated python 3.7.
https://conda-forge.org/docs/user/announcements.html

With the current recommended installation method being:
`conda create -n pygenometracks -c bioconda -c conda-forge pygenometracks python=3.7`

Are there any hopes to support newer versions of python? Thanks!

**Comment by joachimwolff** [comment]
Given our CI, Python 3.8 and 3.9 is supported too: https://github.com/deeptools/pyGenomeTracks/blob/master/azure-pipelines.yml

**Comment by lldelisle** [comment]
Indeed, sorry, the documentation is a bit behind but pygenometracks works with all python versions supported by bioconda.
